apps/src/feedbackBlocks.js

```diff
@@ -67,23 +63,49 @@ var FeedbackBlocks = function(options, missingRequiredBlocks, missingRecommended
       level: options.level,
       blocks: this.xml
     }
-  });
+  };
+
+  this.div = document.createElement('div');
+  this.div.setAttribute('id', 'feedbackBlocksContainer');
+
   this.iframe = document.createElement('iframe');
   this.iframe.setAttribute('id', 'feedbackBlocks');
   this.iframe.setAttribute('allowtransparency', 'true');
+
   this.div.appendChild(this.iframe);
 };
 
 module.exports = FeedbackBlocks;
 
+/**
+ * Generate a url (with params) for the iFrame that's going to hold the
+ * blocks.
+ * TODO(elijah) replace this whole thing with an implementation that
+ * doesn't require an iframe, since we can now have multiple blocklys on
+ * the same page
+ */
+FeedbackBlocks.prototype.readonlyTemplateUrl_ = function () {
+  var params = {
+    app: this.iframeOptions.app,
+    js_locale: this.iframeOptions.options.locale,
+    locale_dir: this.iframeOptions.options.localeDirection
+  };
+
+  // Simple polyfill for $.params
+  var paramString = Object.keys(params).map(function (key) {
+    return key + "=" + encodeURIComponent(params[key]);
+  }).join("&");
+
+  return "/readonly_template?" + paramString;
```

Comment:
This design pattern should be avoided (where the app code is aware of specific URLs in the host environment). When we have needed to do things like this, the URLs are typically supplied to the app as part of `appOptions` (or the milestone response, which may or may not apply here)